R/scenario_description.R

```diff
@@ -0,0 +1,106 @@
+extract_scenario_description <- function(e, path, con) {
+  if (!is.null(e$scenario_description_csv)) {
+    e <- c(list(
```

Comment:
there is no need to assign to `e` and then return it here. The `c` also appears redundant and should be removed

Reply:
Yep, done!